helm_deploy/values-preprod.yaml

```diff
@@ -0,0 +1,18 @@
+---
+# Per environment values which override defaults in prisoner-contact-registry/values.yaml
+
+generic-service:
+  replicaCount: 2
```

Comment:
I think this should be 4 like prod maybe? Something we spotted this week in one of our services